.github/actions/fix-sync/action.yml

```diff
@@ -0,0 +1,120 @@
+name: 'Fix Sync'
+description: 'Ensures proper binaries are in place'
+# This action is required to correct for differences between "gclient sync"
+# on Linux and the expected state on macOS/windows. This requires:
+# 1. Fixing Clang Install (wrong binary)
+# 2. Fixing esbuild (wrong binary)
+# 3. Fixing rustc (wrong binary)
+# 4. Fixing gn (wrong binary)
+# 5. Fix reclient (wrong binary)
+# 6. Fixing dsymutil (wrong binary)
+# 7. Ensuring we are using the correct ninja and adding it to PATH
+# 8. Fixing angle (wrong remote)
+# 9. Install windows toolchain on Windows
+# 10. Fix node binary on Windows
+# 11. Fix rc binary on Windows
+inputs:
+  target-platform:
+    description: 'Target platform, should be linux, win, macos'
```

Comment:
We don't need to change this, just for my understanding - we only need to fix the sync on Mac & Win, right?

Reply:
yes